### PR TITLE
Fix Bug preventing login.

### PR DESCRIPTION
**Pull request by AshitomW**
fixed stupid bug ....

### CODE REVIEW

File: login.js
Comment (AshitomW) [inline login.js:1]: Add some new lines

**Review by AshitomW — COMMENTED** [review]
- Add New Lines

File: login.js
Comment (LkRTom) [inline login.js:1]: Write A New function .

**Review by LkRTom — CHANGES_REQUESTED** [review]
Prettify The Code

File: login.js
Comment (AshitomW) [reply login.js:1]: # Nyet

**Review by LkRTom — APPROVED** [review]
Very Nice , Fraud merge now.

**Pull request merged**